.github/workflows/binary-static-analysis.yml

```diff
@@ -33,21 +31,25 @@ jobs:
         fetch-depth: 2
         submodules: 'recursive'
 
-    - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v2
-      with:
-        python-version: ${{ matrix.python-version }}
+    - name: Init Hermit
+      uses: cashapp/activate-hermit@v1
 
-    - name: Display Python version
+    - name: Install Clang
```

Comment:
Can we share the shared install steps across workflows so we don't have to change a bunch of files when making changes? https://docs.github.com/en/actions/sharing-automations/avoiding-duplication

Reply:
Good idea. I'll look into it.

Reply:
How about I do this in a follow-up PR? This PR is already huge.

Reply:
> How about I do this in a follow-up PR? This PR is already huge.

def better as a separate PR